ci/build.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash -e
+#
+# Build libsnowflakeclient
+#
+set -o pipefail
+THIS_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+source $THIS_DIR/_init.sh
+
+export GIT_COMMIT=`git rev-parse HEAD`
+libsnowflake_ver='0.4.5'
```

Comment:
This can be retrieved by:
```
libsnowflakeclient_ver=$($THIS_DIR/../scripts/build_snowflakeclient.sh -v)
```
Anyway this is not critical at the moment. We can adjust this later.